test/suite/test_compact01.py

```diff
@@ -44,11 +44,8 @@ class test_compact(compact_util, suite_subprocess):
     config = 'leaf_page_max=8KB,key_format=S'
     nentries = 50000
 
-    # The table is a complex object, give it roughly 5 pages per underlying
-    # file.
```

Comment:
Undelete old lines 47-48. It is still true for a SimpleDataSet since a SimpleDataSet has 1 file and 1 x 5 = 5.
Perhaps rewording would be better.
# Give a table 5 pages per underlying file. Note, a SimpleDataSet has only one underlying file.